### PR TITLE
[Workflow] Do not talk about registry

### PR DESCRIPTION
**Pull request by lyrixx**
This is not recommended since https://github.com/symfony/symfony/pull/46000

### CODE REVIEW

File: components/workflow.rst [hunk 1]
```diff
@@ -103,7 +82,6 @@ method to initialize the object property::
 
     // ...
     $blogPost = new BlogPost();
-    $workflow = $registry->get($blogPost);
 
     // initiate workflow
     $workflow->getMarking($blogPost);
```
Comment (OskarStark) [inline components/workflow.rst:87]: $workflow is not defined anymore

File: components/workflow.rst [hunk 2]
```diff
@@ -58,33 +58,12 @@ The ``Workflow`` can now help you to decide what *transitions* (actions) are all
...
 Usage
 -----
 
-When you have configured a ``Registry`` with your workflows,
-you can retrieve a workflow from it and use it as follows::
-
     // ...
     // Consider that $blogPost is in place "draft" by default
     $blogPost = new BlogPost();
-    $workflow = $registry->get($blogPost);
```
Comment (OskarStark) [inline components/workflow.rst:87]: $workflow is not defined anymore

Comment (lyrixx) [reply components/workflow.rst:87]: it is defined in line 55.

File: components/workflow.rst (hunk 1 above)
Comment (lyrixx) [reply components/workflow.rst:87]: it is defined in line 55.

File: components/workflow.rst (hunk 2 above)
Comment (OskarStark) [reply components/workflow.rst:87]: Yes, but shouldn't we show an example of injecting the workflow into a controller action or a service? There is no example for this, right?

Comment (OskarStark) [reply components/workflow.rst:87]: Oh I think it's mentioned here:
```
Read more about the usage of the :doc:`Workflow component </workflow>` inside a Symfony application.
```

**Comment by javiereguiluz** [comment]
Thanks Grégoire.

**Pull request merged**